api/src/paths/project/{projectId}/survey/{surveyId}/observation/submission/get.ts

```diff
@@ -162,7 +159,7 @@ export function getOccurenceSubmission(): RequestHandler {
 
       return res.status(200).json(getOccurrenceSubmissionData);
     } catch (error) {
-      defaultLog.debug({ label: 'getOccurenceSubmission', message: 'error', error });
+      defaultLog.debug({ label: 'getOccurrenceSubmission', message: 'error', error });
```

Comment:
the actual function name on line 94 needs to be changed to fix the spelling as well